src/pages/steckbriefe/index.tsx

```diff
@@ -67,19 +67,21 @@ const SteckbriefeIndex = ({ data }) => {
                     </svg>
                   </div>
                   <h3 className="text-xl font-medium text-slate-900">
-                    {radschnellweg.from} &rarr; {radschnellweg.to}{' '}
+                    {radschnellweg.general.from} &rarr;{' '}
+                    {radschnellweg.general.to}{' '}
                     {/* https://tailwindui.com/components/application-ui/elements/badges */}
                     <span className="inline-flex items-center rounded-full bg-yellow-100 px-3 py-0.5 text-sm font-medium text-yellow-800">
-                      {radschnellweg.state}
+                      {radschnellweg.general.state}
```

Comment:
I guess the state is coming from the Meta JSON

Reply:
It is. The query in this page is just renaming the node to `radschnellwege`

Reply:
Ahh alright, but I wondered, since it does not get displayed ^^

Reply:
But good catch anyways. The `state` attribute is actually not nested inside `general`.